packages/components/organisms/f-checkout/test/specs/component/f-checkout.component.spec.js

```diff
@@ -109,4 +109,8 @@ describe('f-checkout component tests', () => {
         // Assert
         // Waiting for route here, so we can grab redirect url and show form submits.
     });
+
+    it('have switch user link is displayed', () => {
```

Comment:
Could you start this with `should` please.

`should display the switch user link`